ko/admin/db_manage.rst

```diff
@@ -57,9 +57,9 @@ CUBRID 데이터베이스의 볼륨은 크게 영구적 볼륨, 일시적 볼륨
 *   영구적 볼륨 중
  
     *   영구적 데이터를 저장하지만 일시적 데이터도 저장할 수 있는 데이터 볼륨이 있다.
-    *   활성 로그, 보관 로그 및 백그라운드 보관 로그로 세분화할 수 있는 로그 볼륨이 있다.
+    *   활성 로그를 저장하는 로그 볼륨이 있다.
     
-*   일시적 볼륨에는 일시적 데이터만 저장된다.
+*   일시적 볼륨에는 일시적 데이터, 보관 로그 및 백그라운드 보관 로그가 저장된다.
```

Comment:
```suggestion
*   일시적 볼륨 중

    *   임시 데이터를 저장할 수 있는 일시적 볼륨(temporary temp volume)이 있다.
    *   보관 로그를 저장하는 보관 로그(archive log) 볼륨이 있다.
    *   백그라운드에서 로그 보관 작업을 수행할 때 사용되는 백그라운드 보관 로그(background archive log) 볼륨이 있다.
```

Reply:
I suggest as above. I have tried to adhere to the persistent volume description format as much as possible. Please additional opinion on it. (@mhoh3963, @eido5, @wrlawodms, @joohok) @eido5 After the Korean manual review is done, Please apply it to the English manual.

Reply:
The sentence right below refers to another document for more information. I don't think we should explain here what they are, so the below looks enough to me:
```suggestion
*   일시적 볼륨 중
    *   임시 데이터를 저장할 수 있는 일시적 볼륨 (temporary temp volume)이 있다.
    *   보관 로그를 저장을 위한 보관 로그 (archive log) 볼륨과 백그라운드 보관 로그 (background archive log) 볼륨이 있다.
```